test/data.jl

```diff
@@ -71,6 +71,35 @@ module TestData
     sdf6f = view(df6, UInt64[1, 2])
     @test size(sdf6f) == (2,3)
 
+    #test_group("getindex")
+    # ensure indexing column by a single item (by integer/symbol) results in a DataArray
+    v6a = sdf6a[1]
+    @test isa(v6a, DataArray)
+    @test v6a == df6[1, :][1]
+    v6b = sdf6a[:A]
+    @test isa(v6b, DataArray)
+    @test v6b == df6[1, :][:A]
+    # ensure indexing by a collection of columns (by integer/symbol) results in a view
+    # and that indexing by colon results in a DataFrame
+    sdf6g = sdf6a[[1, 2]]
+    @test isa(sdf6g, SubDataFrame)
+    @test isa(sdf6g[:], DataFrame)
+    @test isequal(sdf6g[:], df6[1, [1, 2]])
```

Comment:
Is there a way of comparing `DataFrame`s using `==` in the presence of `NA` without getting `(d == d) == NA` (where `isequal(d, d)` would yield `true`)?